support/Environments/frontera_gcc.sh

```diff
@@ -64,7 +64,7 @@ EOF
         echo "Installing Blaze..."
```

Comment:
I've added Blaze 3.8 to Wheeler as a module. Could you update the two Wheeler envs and verify that works?

Reply:
Thanks! I'll check that and confirm here once I've made sure everything builds and tests alright.